android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -107,6 +107,7 @@ public void run() {
       emptyGroupIdQueue();
       mSecondScanComplete = true;
       mSwipeRefreshWidget.setRefreshing(false);
+      mScanningAnimationTextView.setAlpha(0f);
```

Comment:
Would it make sense to call this everywhere we already call mSwipeRefreshWidget.setRefreshing(false)?